OpenShift/helm/nexus-repository-manager/Chart.yaml

```diff
@@ -1,13 +1,23 @@
-name: nexus-repository-manager
-version: 0.1
-description: A reference OpenShift deployment for Nexus Repository Manager
+apiVersion: v1
+name: sonatype-nexus
+version: 0.1.0
+appVersion: 3.20.1-01
+description: Sonatype Nexus is an open source repository manager
 keywords:
-- quickstart
-  ci
-  repository-manager
-  nexus3
-  nexus
-  sonatype
-home: https://www.sonatype.com/
+  - artifacts
+  - dependency
+  - management
+  - sonatype
+  - nexus
+  - repository
+home: https://www.sonatype.com/nexus-repository-oss
+icon: https://www.sonatype.com/hubfs/2019%20Horizontal%20Product%20logo/horizontal%20Repo%20/Repo/NexusRepo_horiz.svg
```

Comment:
is there a better "permalink" url we should be using for this icon?

Reply:
I've asked around and that was they best I could get  :(